CHANGELOG.md

```diff
@@ -6,6 +6,53 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - getPreferencesState provides the default value for `useTokenDetection` from the PreferencesController state.
+  - getNetworkState provides the default value for `chainId` from the NetworkController state.
+  - onTokenListStateChange listener triggers a detect action whenever the TokenListController finishes fetching a token list.
+- **BREAKING:** Expose `chainId` to AssetsContractController and add check whether current network supports token detection ([#809](https://github.com/MetaMask/controllers/pull/809))
+  - Consumers will need to pass listener method `onNetworkStateChange` in `AssetsContractController` constructor options object. This method should be called with network/provider details when network changes occur.
+- Add `onCollectibleAdded` event handler to the CollectiblesController constructor ([#814](https://github.com/MetaMask/controllers/pull/814))
+  - This event handler was added to allow us to capture metrics.
+
+### Changed
+- **BREAKING:** Change `addTokens` to `addDetectedTokens` on the TokenDetectionController constructor ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - We are no longer automatically adding detected tokens to the wallet. This will provide users with the ability to manually import or ignore detected tokens.
+- **BREAKING:** Rename `useStaticTokenList` to `useTokenDetection` on the `PreferencesController` constructor options object and set the value to be true by default ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - Token detection will now be enabled by default.
+- **BREAKING:** Append phishfort blocklist to list used by PhishingController ([#715](https://github.com/MetaMask/controllers/pull/715))
+  - The `test` method on `PhishingController` no longer returns a boolean, it now returns an object matching the `EthPhishingDetectResult` interface (defined in `PhishingController.ts`).
+  - Designs may need to be updated to account for the fact that sites may now be blocked by Phishfort. We should ensure users are not directed to the `eth-phishing-detect` repository to dispute Phishfort blocks, because we will not be able to help them.
+- **BREAKING:** Rename `convertPriceToDecimal` method name to `convertHexToDecimal` ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - Consumers of this method must update the function name wherever it is used.
+- Update TokensController to support detected tokens ([#808](https://github.com/MetaMask/controllers/pull/808))
+  - Introduce `detectedTokens` to the config object to track detected tokens, which is updated by calling `addDetectedTokens` whenever TokenDetectionController detects new tokens.
+  - Added `fetchTokenMetadata` private method to fetch token metadata whenever adding individual tokens. This is currently used to populate aggregator information.
```

Comment:
This isn't private, nor is it a method. Maybe say "Added `fetchTokenMetadata` utility function to...."?

Reply:
updated here: https://github.com/MetaMask/controllers/pull/848/commits/8f0b66e74d974e5e1da4d250f99b289e2eff70c3

Reply:
Probably not a good idea on my part to name the method within the class to be the same as the utility function that it's importing. There is actually a private method that lives in the TokensController controller named `fetchTokenMetadata`. :sweat_smile:

Reply:
Ah, okay!